torchgen/gen.py

```diff
@@ -481,7 +482,14 @@ class RegisterSchema:
     def __call__(self, f: NativeFunction) -> Optional[str]:
         if not self.selector.is_native_function_selected(f):
             return None
-        return f"m.def({cpp_string(str(f.func))});\n"
+        if len(f.tags) == 0:
+            return f"m.def({cpp_string(str(f.func))});\n"
```

Comment:
Also small nit, but you can probably just avoid adding a special case in the codegen for `len(f.tags) == 0`, since we'll just stamp out an empty set into `m.def()`.